master/docs/manual/installation/installation.rst

```diff
@@ -75,7 +75,9 @@ If you cannot or do not wish to install the buildbot into a site-wide location l
 Running Buildbot's Tests (optional)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you wish, you can run the buildbot unit test suite.
+If you wish, you can run the buildbot unit test suite. You must not be using a python wheels packaged version 
```

Comment:
Buildbot is spelled as `Buildbot`.

We have "one sentence per line policy" (http://docs.buildbot.net/latest/developer/style.html)